var/spack/repos/builtin/packages/py-dask-ml/package.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyDaskMl(Package):
+    """Scalable Machine Learning with Dask."""
+
+    homepage = "https://ml.dask.org/"
+    url      = "https://pypi.io/packages/source/d/dask-ml/dask-ml-1.8.0.tar.gz"
```

Comment:
```suggestion
    pypi     = "dask-ml/dask-ml-1.8.0.tar.gz"
```

Reply:
I modified it to `PythonPackage` format.